tools/cloud_functions/gcs_event_based_ingest/gcs_ocn_bq_ingest/common/utils.py

```diff
@@ -40,96 +42,157 @@
 
 from . import constants  # pylint: disable=no-name-in-module,import-error
 from . import exceptions  # pylint: disable=no-name-in-module,import-error
+from . import logging  # pylint: disable=no-name-in-module,import-error
 
 
 def external_query(  # pylint: disable=too-many-arguments
         gcs_client: storage.Client, bq_client: bigquery.Client, gsurl: str,
-        query: str, dest_table_ref: bigquery.TableReference, job_id: str):
+        query: str, job_id: str, table: bigquery.TableReference):
     """Load from query over external table from GCS.
 
     This hinges on a SQL query defined in GCS at _config/*.sql and
-    an external table definition _config/external.json (otherwise will assume
+    an external table definition
+    _config/{constants.BQ_EXTERNAL_TABLE_CONFIG_FILENAME} (otherwise will assume
     CSV external table)
```

Comment:
Shouldn't this be PARQUET instead of CSV?

Reply:
Good catch!